server/controllers/appointment.js

```diff
@@ -0,0 +1,48 @@
+const asyncHandler =  require('express-async-handler');
+const validatorResults = require('express-validator');
+const Appointment = require('../models/Appointment');
+const Meeting = require('../models/Meeting');
+const moment = require('moment-timezone');
+//CREATE /appointment -> Create an appointment
+//meeting_id, name, email, time (datetime), timezone
+//GET /appointments -> list of appointments for logged in user
+
+// @route  POST api/appointment
+// @des    Creates appointment
+// @access Public
+exports.createAppointment = asyncHandler(async (req, res) => {
+    const { name, email, time } = req.body;
+    const meeting = await Meeting.findOne({ meetingId: req.params.user_id });
```

Comment:
The user id is the meeting id?

Reply:
The last comment kinda confused me on this one. It seemed as if you were talking about this meetingId being the same as userId which shouldn't happen. So i references the meetingId from the Meeting model.

Reply:
Disregard last comment, this code here stood out to me because `meetingId` to me is the id of the meeting and shouldn't be the user id, conventionally this isn't a great name choice. If the meeting id is just a user ID then we should just rename `meetingId` to `userId`

Reply:
Otherwise, the current change you did I approved and something we can touch up in the future

Reply:
Very well then.